cookiecutter/{{cookiecutter.organization}}-richie-site-factory/Makefile

```diff
@@ -56,6 +56,7 @@ bootstrap:  ## install development dependencies
 .PHONY: bootstrap
 
 add-site: ## add a new site to the site factory
+	@mkdir -p .cookiecutter_replay
 	@docker run --rm -it -e LC_ALL=C.UTF-8 \
```

Comment:
Oh ok. Then maybe add a `. git` keep file in the `.cookiecutter_replay` directory?

Reply:
With that, I think we don't need the line on the Mafile.

Reply:
@igobranco do you want to change your PR to add the .gitkeep file or do you want me to open another PR?

Reply:
Updated the PR.